splinterd/src/config/builder.rs

```diff
@@ -228,6 +228,14 @@ impl ConfigBuilder {
                     None => None,
                 })
                 .ok_or_else(|| ConfigError::MissingValue("registry file".to_string()))?,
+            registries: self
```

Comment:
typo in commit message `URIs with be supported` -> `URIs will be supported`

Reply:
Fixed